ntoskrnl/io/iomgr/file.c

```diff
@@ -1262,6 +1262,7 @@ IopParseDevice(IN PVOID ParseObject,
                 FileBasicInfo = ExAllocatePoolWithTag(NonPagedPool,
                                                       sizeof(*FileBasicInfo),
                                                       TAG_IO);
+                RtlZeroMemory(FileBasicInfo, sizeof(*FileBasicInfo));
```

Comment:
Maybe, but not at this line...

Reply:
Which line I should use RtlZeroMemory?

Reply:
Inside the `if (FileBasicInfo)` block, since here it could be NULL.

Reply:
I changed, thanks.